src/services/auth/oauth1/types.ts

```diff
@@ -0,0 +1,66 @@
+import { AffectedRecords, OptionsWithRql, PagedResult } from '../../types';
+
+export interface AuthOauth1Service {
+  /**
+   * Generate an SSO token with OAuth 1 authentication
+   *
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | Everyone can use this endpoint
+   * @see https://swagger.extrahorizon.com/swagger-ui/?url=https://swagger.extrahorizon.com/auth-service/2.0.4-dev/openapi.yaml#/SSO/post_oauth1_ssoTokens_generate
+   * @throws {ApplicationNotAuthenticatedError}
+   * @throws {UserNotAuthenticatedError}
+   */
+  generateSsoToken(): Promise<SsoToken>;
+  /**
+   * Consume an SSO token to get OAuth 1 tokens
+   *
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | Everyone can use this endpoint
+   * @see https://swagger.extrahorizon.com/swagger-ui/?url=https://swagger.extrahorizon.com/auth-service/2.0.4-dev/openapi.yaml#/SSO/post_oauth1_ssoTokens_generate
```

Comment:
I think all of these links are outdated. 🤔

Reply:
Yes, first wanted to do the restructuring. I will tackle the inline docs including links next